app-modules/knowledge-base/src/Models/KnowledgeBaseItem.php

```diff
@@ -71,6 +71,7 @@ class KnowledgeBaseItem extends BaseModel implements Auditable, HasMedia, HasTag
     protected $fillable = [
         'article_details',
         'category_id',
+        'portal_view_count',
```

Comment:
Let's actually make the choice not to have this fillable. Since we should only ever be incrementing this by 1.

That will mean you will need to increment the value and save it. But that technically is safer

Reply:
Done